### DISCUSSION TITLE
next/image slow request time for optimized images

### DISCUSSION THREAD

**Discussion by jfarris587** [vercel/next.js#26877] [Help]
Using the `next/image` component, images are served as `webp` with an optimized size. However, when using locally served images, the request time is very slow. Often taking several seconds to load. In fact a normal `<img>` tag with an unoptimized image `src` that is several Mb larger is _faster_

The point of `next/image` is to speed up size speed and optimization, but this usage makes it difficult to justify - the result is a UX that looks and feels much slower to the user, even thought the images have an 'optimized size'

Is this known? Is this a problem with using `next/image` for locally served images instead of a CDN? What suggestions are there?

See below. The first requested image is a normal `<img>` that is returned instantly with larger size. the last image is a lazy loaded image that is loaded after scrolling. **_It took 3 seconds to load the image even though its a simple kb file_**

<img width="757" alt="Screen Shot 2021-07-02 at 12 20 59 PM" src="https://user-images.githubusercontent.com/31263423/124303166-1b6d6780-db30-11eb-9750-c4d9b9a8c696.png">

**Comment by sebagruia** [comment]
I'm facing the same issue.

**Reply by wwwdepot** [reply]
related to https://github.com/vercel/next.js/issues/23637
